src/sources/gdal.jl

```diff
@@ -391,3 +391,66 @@ build_geotransform(lat, lon) = begin
     gt[GDAL_NS_RES] = step(lat)
     return gt
 end
+
+function GeoArray(dataset::AG.Dataset, key = nothing;
+                  usercrs = nothing,
```

Comment:
I like to (kind of) follow the BlueStyle guide - with no spaces around keyword args `=`.

Reply:
In function definitions only or also in function calls?

Reply:
I think both

Reply:
done